### PR TITLE
Target .NET Standard 2.0

### LINKED ISSUE #204: Add target .NET Standard 2.0
This is a spinoff of issue #199 

As pointed out in https://github.com/hedgehogqa/fsharp-hedgehog/issues/199#issuecomment-653670228, one advantage of targeting .NET Standard 2.0 even when also targeting .NET Standard 1.6 is because there can be fewer dependencies for users that are able to consume the version that targets .NET Standard 2.0.

### PR DESCRIPTION
**Pull request by bender2k14**
(Will eventually...) Fix #204

I am not very experienced with paket.  I assumed that I should add `netstandard2.0` [here](https://github.com/hedgehogqa/fsharp-hedgehog/compare/master...bender2k14:also_target_netstandard2.0?expand=1#diff-b27553841b4413a24ed1b9aff865105bL10-R10).  Then I deleted the `paket.lock` file and recreated it by executing `paket install`.

That all seemed to work, but then building the code fails because paket cannot parse the lock file it created.
```
Determining projects to restore...
Performance:
 - Runtime: 331 milliseconds
Paket failed with
-> Error during parsing of 'C:\Code\ThirdParty\fsharp-hedgehog\paket.lock'.
-> Could not parse 'framework: restriction: || (== netstandard1.6) (&& (== netstandard2.0) (>= netcoreapp5.0))' as key/value pairs.
-> invalid delimiter at position 22
```

Can someone give me a debugging suggestion or simply tell me how I am using paket wrong?

### CODE REVIEW

**Comment by moodmosaic** [comment]
Thanks for the pull request!

Off the top of my head, I don't know what the syntax is... Also, I wonder if we could use just `dotnet` and drop `paket` entirely. I like Paket a lot, just trying to simplify things when possible.

If you want to take a crack on using just `dotnet` to restore the packages (and build the NuGet package) I'm all in. Otherwise I can look at the Paket syntax on Monday, the earliest.

File: paket.dependencies [hunk 1]
```diff
@@ -7,7 +7,7 @@ storage: none
 
 group Build
 
-    framework: netstandard1.6
+    framework: netstandard1.6, netstandard2.0
```
[the remark this answers is not in the document]
Comment (moodmosaic) [reply paket.dependencies:10]: So I just updated to the [current Paket version (5.247.4)](https://github.com/fsprojects/Paket/releases/tag/5.247.4) and now I can build the project but can't run the tests. 😕

[the remark this answers is not in the document]
Comment (moodmosaic) [reply paket.dependencies:10]: Yeah, I just can't get this to work (I updated both the executable and the Paket.restore.targets file). Would you like to close this PR in favor of #207?

[the remark this answers is not in the document]
Comment (bender2k14) [reply paket.dependencies:10]: I would prefer to come back to this after #206 (with current draft PR #207) as been resolved.  I think it is fine to leave this open in the meantime.

**Comment by bender2k14** [comment]
Will open another PR to accomplish this soon now that paket has been removed.

**Pull request closed without merging**